pandas/core/arrays/numpy_.py

```diff
@@ -436,6 +436,11 @@ def skew(self, axis=None, dtype=None, out=None, keepdims=False, skipna=True):
     # ------------------------------------------------------------------------
     # Additional Methods
 
+    def astype(self, dtype, copy=True):
+        if dtype is self.dtype:
```

Comment:
Not a singleton, no

Reply:
does this require action?  AFAICT the fact that this is needed is an artifact of the way we are patching PandasArray._typ in these tests

Reply:
I think so. Otherwise the implementation is incorrect since `PandasArray.astype(PandasDtype("int64"))` will raise

```python
In [6]: arr = pd.core.arrays.PandasArray(np.array([1, 2]))

In [7]: arr.astype(pd.core.arrays.numpy_.PandasDtype("float64"))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-eb4babcd3e70> in <module>
----> 1 arr.astype(pd.core.arrays.numpy_.PandasDtype("float64"))

~/sandbox/pandas/pandas/core/arrays/numpy_.py in astype(self, dtype, copy)
    440         if dtype is self.dtype:
    441             return self.copy() if copy else self
--> 442         return super().astype(dtype, copy=copy)
    443
    444     def to_numpy(

~/sandbox/pandas/pandas/core/arrays/base.py in astype(self, dtype, copy)
    432             NumPy ndarray with 'dtype' for its dtype.
    433         """
--> 434         return np.array(self, dtype=dtype, copy=copy)
    435
    436     def isna(self) -> ArrayLike:

TypeError: data type not understood
```

Reply:
You're right, but that isnt any different from the status quo

Reply:
updated to revert the `astype` change and xfail the affected tests